strapi/strapi
# chore(databse): clean up db naming methods

### What does it do?

- removes a few methods that were confusing and not needed
- pass options through to naming methods

### Why is it needed?

Describe the issue you are solving.

### How to test it?

- the model names for all these methods are tested with unit tests so as long as they pass it's fine
- as a manual test, you could fill a strapi project with lots of data in v5/main, then switch to this branch and start up, and all your data should still be there

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
